### PR TITLE
Bug/gb ui 56 oncomplete called before finished

### PR DESCRIPTION
**Pull request by rfreitas**
Fix for issue: https://github.com/onfido/onfido-sdk-ui/issues/56

### CODE REVIEW

File: src/store/reducers/captures.js [hunk 1]
```diff
@@ -6,23 +6,30 @@ const initialState = {
...
+)
+
 export function captures(state = initialState, action) {
-  const { payload } = action
+  const payload = action.payload || {}
+  const captureType = payload.method? payload.method : payload
+  const captures = state[captureType]
+
+  const newStateWithCaptureState = newCaptureState => objectAssign({}, state, { [captureType]: newCaptureState })
+  const changeCapturesStateBinded = changeCapturesState.bind(this, captures)
```
Comment (stephencookdev) [inline src/store/reducers/captures.js]: `Bound` rather than `Binded`, I think?

**Comment by stephencookdev** [comment]
LGTM :+1:

File: src/store/reducers/captures.js (hunk 1 above)
Comment (rfreitas) [reply src/store/reducers/captures.js]: Yes, actually `bonded`, no?

Comment (rfreitas) [reply src/store/reducers/captures.js]: maybe not..

**Pull request merged**